collect_app/src/main/java/org/odk/collect/android/database/DatabaseFormColumns.java

```diff
@@ -0,0 +1,38 @@
+package org.odk.collect.android.database;
+
+import android.provider.BaseColumns;
+
+/**
+ * Columns for the Forms table.
+ */
+public final class DatabaseFormColumns implements BaseColumns {
```

Comment:
It would have been nice to also have a `FormsProviderAPI.FormsColumns` but it doesn't seem worth it unless we want to rename a column in the DB (but still return the same thing from the content provider).